src/utils/bets.ts

```diff
@@ -6,3 +10,37 @@ export const incrementPage = (page: number, maxPages: number) =>
 
 export const decrementPage = (page: number, maxPages: number) =>
   page - 1 < 1 || page - 1 > maxPages ? maxPages : page - 1;
+
+const getBetStatus = (
+  bet: Bet,
+  signedBetData: SignedBetDataResponse
+): BetStatus => {
+  const hasResult =
+    signedBetData.winningPropositionId || signedBetData.marketResultAdded;
+  if (!hasResult && !bet.settled) return "PENDING";
+  if (hasResult && !bet.settled) return "RESULTED";
+  if (hasResult && bet.settled) return "SETTLED";
+  throw new Error("Invalid bet status");
+};
+
+export const handleBetHistory = (
```

Comment:
this looks like formatting to me -- imo it should be in `formatting.ts` as `formatBetHistory`